opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -95,14 +95,141 @@ message InstrumentationLibraryMetrics {
 //                        |+-----+                    |
 //                        +---------------------------+
 //
+// Each distinct type of DataPoint represents the output of a specific
+// aggregation function, the result of applying the DataPoint's
+// associated function of to one or more measurements.
+//
 // All DataPoint types have three common fields:
-// - Labels zero or more key-value pairs associated with the data point.
-// - StartTimeUnixNano MUST be set to the start of the interval when the data's
-//   type includes an AggregationTemporality. This field is not set otherwise.
-// - TimeUnixNano MUST be set to:
-//   - the moment when an aggregation is reported (independent of the
-//     aggregation temporality).
-//   - the instantaneous time of the event.
+// - Attributes includes key-value pairs associated with the data point
+// - TimeUnixNano is required, set to the end time of the aggregation
+// - StartTimeUnixNano is optional, but strongly encouraged for DataPoints
+//   having an AggregationTemporality field, as discussed below.
+//
+// Both TimeUnixNano and StartTimeUnixNano values are expressed as
+// UNIX Epoch time in nanoseconds since 00:00:00 UTC on 1 January 1970.
+//
+// # TimeUnixNano
+//
+// This field is required, having consistent interpretation across
+// DataPoint types.  TimeUnixNano is the moment corresponding to when
+// the data point's aggregate value was captured.
+//
+// Data points with/ the 0 value for TimeUnixNano SHOULD be rejected
+// by consumers.
+//
+// # StartTimeUnixNano
+//
+// StartTimeUnixNano in general allows detecting when a sequence of
+// observations is unbroken.  StartTimeUnixNano is set to the
+// beginning of an unbroken sequence of observations (all point kinds
+// except with delta aggregation temporality) or the end of an
+// unbroken sequence of observations (point kinds with delta
+// temporality).
+//
+// This field indicates to consumers the start time for points with
+// cumulative and delta AggregationTemporality, and it should be
+// included whenever possible to support correct rate calculation.
+// Although it may be omitted when the start time is truly unknown,
+// setting StartTimeUnixNano is strongly encouraged.
+//
+// There are two reasons that streams with AggregationTemporality are
+// reset: (1) a process restarts, and (2) the process "forgets" the stream
+// and then re-initializes it.  StartTimeUnixNano is critical for
+// calculating correct rates for Monotonic Sum, Histogram, and Summary
```

Comment:
Summary does not yet have `AggregationTemporality` and hence not "critical" to know StartTimeUnixNano. Even if you know it is little to be doing with that.

Reply:
The `sum` and `count` fields of a Summary have cumulative interpretation, which is why I say it is strongly encouraged to have a start time--for calculating rates from these (and overlaps). Does that sound right?

Reply:
Ok, based on https://github.com/OpenObservability/OpenMetrics/blob/main/specification/OpenMetrics.md#summary, we need to update our definition of Summary and say that count and sum have "cumulative" temporality (because we right now don't have a temporality in the Summary definition) and quantiles do not respect that? Am I wrong?

Reply:
+1.   Summary has no ability to define aggregation temporality (didn't notice this) so we shouldn't specify that it can, it's always cumulative.

Reply:
See if the new text covers this? I've got 
```
// ## StartTimeUnixNano: Summaries
//
// Summary points are treated the same as for cumulative aggregation
// temporality, by convention, allowing their `_sum` and `_count` fields
// to be used for rate calculations.  StartTimeUnixNano should be set to the
// starting timestamp in an unbroken sequence of observations.
```